providers/build.info

```diff
@@ -1 +1,15 @@
 SUBDIRS=common default
+
+IF[{- !$disabled{fips} -}]
+  SUBDIRS=fips
+  MODULES=fips
+  IF[{- defined $target{shared_defflag} -}]
+    SHARED_SOURCE[fips]=fips.ld
+    GENERATE[fips.ld]=provider.num
```

Comment:
That should be

```
    GENERATE[fips.ld]=../util/providers.num
```

There's no reason to make multiple copies of it...

Reply:
("it" being that ordinals file)

Reply:
Fixed